### PR TITLE
[MMDS][CDAP-12704] Adds more details to Model view

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds a `CollapsibleWrapper` to automatically show text (in case of mmds features and directives) in a popover when there is no space
- Fixes the "Add a Model" popover to open the model popover while trying to add a model to an experiment
- Removes the "Edit" link to edit an experiment (Right now user can only change the description and it doesn't provide that much of functionality to have it).

JIRA: https://issues.cask.co/browse/CDAP-12704

### CODE REVIEW

File: cdap-ui/app/cdap/components/Experiments/store/ActionCreator.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ function getSplitsInExperiment(experimentId) {
 
 function getModelStatus(experimentId, modelId) {
   myExperimentsApi
-    .getModelStatus({
+    .pollModelStatus({
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Experiments/store/ActionCreator.js]: Will this poll function be stopped anywhere?

**Comment by tonybach** [comment]
Might not be related to this PR, but when I created an experiment and hadn't added a model yet, it still says 1 model in the header and in the pagination.

screen shot 2018-02-12 at 11 30 27 am

File: cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js [hunk 2]
```diff
@@ -129,16 +132,80 @@ const renderMetrics = (newHeaders, model) => {
...
+  let endTime = nowInSeconds;
+  if (splitId) {
+    let {routerServerUrl, routerServerPort} = window.CDAP_CONFIG.cdap;
+    let protocol = window.CDAP_CONFIG.sslEnabled ? 'https' : 'http';
+    if (routerServerUrl === '127.0.0.1') {
+      routerServerUrl = 'localhost';
+    }
+    let hostPort = `${protocol}://${routerServerUrl}:${routerServerPort}`;
+    let baseUrl = `/v3/namespaces/${getCurrentNamespace()}/apps/ModelManagementApp/spark/ModelManagerService/logs`;
+    let queryParams = `?filter=MDC:experiment="${experimentId}"%20AND%20MDC:model=${model.id}&start=${startTime}&end=${endTime}`;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js]: is it possible to just use `encodeURI`? would that mess up the experimentID AND MDC?

**Comment by ajainarayanan** [comment]
@tonybach The error in count for number of models you are seeing is a backend issue. @CuriousVini is working on it.

File: cdap-ui/app/cdap/components/Experiments/store/ActionCreator.js [hunk 3]
```diff
@@ -168,21 +168,21 @@ function getSplitsInExperiment(experimentId) {
...
     })
     .subscribe(modelStatus => {
-      experimentDetailsStore.dispatch({
-        type: EXPERIMENTDETAILACTIONS.SET_MODEL_STATUS,
-        payload: {
-          modelId,
-          modelStatus
-        }
-      });
+    experimentDetailsStore.dispatch({
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Experiments/store/ActionCreator.js]: Did you change the indentation of this block by accident?

File: cdap-ui/app/cdap/components/Experiments/DetailedView/index.js [hunk 4]
```diff
@@ -41,6 +43,8 @@ export default class ExperimentDetails extends Component {
     location: PropTypes.object
   };
 
+  modelStatusObsertables = [];
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Experiments/DetailedView/index.js]: nit: you mean `Observables`? 😄

**Review by tonybach — APPROVED** [review]
Code LGTM 👍

**Comment by ajainarayanan** [comment]
Demoed it @leacd . Merging.

**Comment by leacd** [comment]
LGTM

**Pull request merged**